notifier/main.ts

```diff
@@ -1,12 +1,67 @@
 import * as path from "std/path/mod.ts";
-import { Application } from "oak/mod.ts";
-import { Bot } from "grammy/mod.ts";
+import { Application, Router } from "oak/mod.ts";
+import { Bot, Context, webhookCallback } from "grammy/mod.ts";
+import { FileFlavor, hydrateFiles } from "grammy_files/mod.ts";
```

Comment:
```suggestion
import { Bot, type Context, webhookCallback } from "grammy/mod.ts";
import { type FileFlavor, hydrateFiles } from "grammy_files/mod.ts";
```

Reply:
It doesn’t make a difference when compiled, but OK.